internal/tools/start_test.go

```diff
@@ -0,0 +1,71 @@
+package tools
```

Comment:
I would vote for introducing ginkgo testing framework (should not be part of this PR)